fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1091,6 +1156,20 @@ ACTOR Future<Void> serveTLogInterface_PassivelyPull(
 				self->addActors.send(tLogCommit(logData->tlogGroupData, req, logData));
 			}
 		}
+		when(TLogPeekRequest req = waitNext(tli.peek.getFuture())) {
+			TraceEvent("TLogPeekReq")
+			    .detail("BeginVersion", req.beginVersion)
+			    .detail("StorageTeam", req.storageTeamID)
+			    .detail("Tag", req.tag.toString());
+			auto tlogGroup = activeGeneration.find(req.storageTeamID);
+			TEST(tlogGroup == activeGeneration.end()); // TLog peek: group not found
+			if (tlogGroup == activeGeneration.end()) {
+				req.reply.sendError(tlog_group_not_found());
+				continue;
+			}
+			Reference<LogGenerationData> logData = tlogGroup->second;
```

Comment:
tLogPeekMessages can only take logData parameter. tlogGroupData can be avoided.

Reply:
Done.